Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,27 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="Scuba diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "Hey [player], have you heard about scuba diving?"
+    m 3eub "After immersion in water, you can see a wonderful underwater world full of diverse marine life."
```

Comment:
If you have the same expression multiple times in a row, you only need to specify the first time. 

Could you include an additional line or two explaining what scuba diving actually is before getting into what it's like?